sdk/textanalytics/ai-text-analytics/src/generated/models/parameters.ts

```diff
@@ -1,48 +1,46 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Autorest.Typescript needs to support the license-header flag. 
Azure/autorest.typescript/issues/555